python/ray/worker.py

```diff
@@ -1834,6 +1835,7 @@ def connect(node,
     # Create an object store client.
     worker.plasma_client = thread_safe_client(
         plasma.connect(node.plasma_store_socket_name, None, 0, 300))
+    driver_id_str = _random_string()
```

Comment:
can we use the length= parameter of _random_string and then not do the indexing below? That seems cleaner to me.